R/mod_nhs_download.R

```diff
@@ -30,7 +28,9 @@ mod_nhs_download_ui <- function(id) {
               fill = "none"
             ),
             tags$path(
-              d = "M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z"
+              d = "M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 
+                   1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 
+                   1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z"
```

Comment:
Just make it easier to take in when in RStudio, by limiting line length to ~80-90 characters. The line breaks and spaces have no effect on the final HTML produced (multiple whitespace is reduced to a single whitespace).